packages/atomic/cypress/integration/sort-dropdown.cypress.ts

```diff
@@ -70,14 +67,42 @@ describe('Sort Dropdown Component', () => {
     );
   });
 
-  it(`when a criteria is not valid
-    should render an error`, () => {
+  it('Should reflect selected sort on URL', () => {
+    selectOption(criteria[1]);
+    const urlHash = `sortCriteria=${encodeURIComponent(criteria[1])}`;
+    cy.url().should('include', urlHash);
+  });
+});
+
+describe('When sort with invalid criteria option', () => {
+  it('Should render an error when <atomic-sort-criteria> is invalid', () => {
     setUpPage(`
-      <atomic-sort-dropdown>
-        <atomic-sort-criteria</atomic-sort-criteria>
-      </atomic-sort-dropdown>
-    `);
-    cy.wait(500);
+        <atomic-sort-dropdown>
+          <atomic-sort-criteria</atomic-sort-criteria>
```

Comment:
What's the difference between this test and the one on line 89-91 ? It seems very similar ? 🤔 

Both has no `criteria` defined.

Reply:
ha, really, I found it different. But nevermind, I will modify, to have the test without `<atomic-sort-dropdown>` then